apisix/plugins/limit-req.lua

```diff
@@ -68,8 +104,23 @@ end
 
 
 local function create_limit_obj(conf)
-    core.log.info("create new limit-req plugin instance")
-    return limit_req_new("plugin-limit-req", conf.rate, conf.burst)
+    if conf.policy == "local" then
+        core.log.info("create new limit-req plugin instance")
+        return limit_req_new("plugin-limit-req", conf.rate, conf.burst)
+    elseif conf.policy == "redis" then
+
```

Comment:
No need blank here

Reply:
https://github.com/apache/apisix/blob/master/CODE_STYLE.md#blank-line
But I noticed multiple 'elseif' in the code_style that require blank lines. Are you sure we should remove these blanks? By the way, I also think it doesn't look good that way.

Reply:
I think it should like this 
```
    if conf.policy == "local" then
        core.log.info("create new limit-req plugin instance")
        return limit_req_new("plugin-limit-req", conf.rate, conf.burst)

    elseif conf.policy == "redis" then
        core.log.info("create new limit-req redis plugin instance")
        return redis_single_new("plugin-limit-req", conf, conf.rate, conf.burst)

    elseif conf.policy == "redis-cluster" then
        core.log.info("create new limit-req redis-cluster plugin instance")
        return redis_cluster_new("plugin-limit-req", conf, conf.rate, conf.burst)
    
```

Reply:
yeah, u are right.